index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter Clone</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="menu">
+        <div class="menu-structure">
+          <ion-icon name="logo-twitter" class="icon main-icon"></ion-icon>
+          <div class="menu-item">
+            <ion-icon name="home-outline" class="icon"></ion-icon>
+            <h2>Home</h2>
+          </div>
+          <div class="menu-item">
+            <ion-icon name="globe-outline" class="icon"></ion-icon>
+            <h2>Explore</h2>
+          </div>
+          <div class="menu-item">
+            <ion-icon name="notifications-outline" class="icon"></ion-icon>
+            <h2>Notifications</h2>
+          </div>
+          <div class="menu-item">
+            <ion-icon name="mail-outline" class="icon"></ion-icon>
+            <h2>Messages</h2>
+          </div>
+          <div class="menu-item">
+            <ion-icon name="bookmark-outline" class="icon"></ion-icon>
+            <h2>Bookmarks</h2>
+          </div>
+          <div class="menu-item">
+            <ion-icon name="list-outline" class="icon"></ion-icon>
+            <h2>Lists</h2>
+          </div>
+          <div class="menu-item">
+            <ion-icon name="person-circle-outline" class="icon"></ion-icon>
+            <h2>Profile</h2>
+          </div>
+          <div class="menu-item">
+            <ion-icon
+              name="ellipsis-horizontal-circle-outline"
+              class="icon"
+            ></ion-icon>
+            <h2>More</h2>
+          </div>
+          <button class="tweet-btn">Tweet</button>
+        </div>
+        <div class="user-btn">
+          <img
+            src="https://randomuser.me/api/portraits/men/40.jpg"
+            alt="user"
+          />
+          <div class="user-name">
+            <p class="user-text">Kevin</p>
+            <p class="user-tag">@Kevlord123</p>
+          </div>
+          <ion-icon
+            name="ellipsis-horizontal-outline"
+            class="user-icon"
+          ></ion-icon>
+        </div>
+      </div>
+
+      <div class="main"></div>
+      <div class="newsfeed"></div>
```

Comment:
loved this entire section especially the search bar with being able to type inside. Was clicking all around the site